src/MainNav/MainNav.tsx

```diff
@@ -3,7 +3,11 @@ import NavSearch from '../NavSearch/NavSearch'
 import ParkList from '../ParkList/ParkList'
 import parkfinderLogo from './landscape.png'
 
-function MainNav() {
+interface FuncProps {
```

Comment:
Do we want to consider moving all the interfaces into one file? I'm not sure which is easier/better developer empathy? (Very much NOT a priority right now!)